sbws/core/generate.py

```diff
@@ -106,7 +111,7 @@ def main(args, conf):
     bw_file = V3BWFile.from_results(results, state_fpath, args.scale_constant,
                                     scaling_method,
                                     torflow_cap=args.torflow_bw_margin,
-                                    torflow_round_digs=args.torflow_round_digs,
+                                    round_digs=args.round_digs,
```

Comment:
This change should work, because --round-digs is the first long option argument in the list:
https://docs.python.org/3/library/argparse.html#dest

Reply:
This change should work without any changes to the arguments.

Reply:
Yes, but then you suggest but then we're never using --torflow-round-digs and we can't cause it won't be in the args Namespace.
You said in a previous comment:
> When --torflow-round-digs is passed to sbws, sbws should do what it did in 1.0.

Reply:
I don't understand what the problem is here.

If you revert new lines 57-63 to old lines 57-58 in master, then:
1. args.round_digs should exist:
https://docs.python.org/3/library/argparse.html#dest
2. when --torflow-round-digs is passed to sbws, sbws will do what it did in 1.0

Can you show me a test where this does not work?

Reply:
> I don't understand what the problem is here.
> 
> If you revert new lines 57-63 to old lines 57-58 in master, then:
> 
>     1. args.round_digs should exist:
>        https://docs.python.org/3/library/argparse.html#dest
correct
>     2. when --torflow-round-digs is passed to sbws, sbws will do what it did in 1.0

no, --torflow-round-digs won't be in the namespace and it's not passed in V3BWFile.from_file in your suggestion.
> 
> Can you show me a test where this does not work?

I can create a test, but if you just checkout current master and run 'sbws generate`, you get:
`AttributeError: 'Namespace' object has no attribute 'torflow_round_digs'`
If you try `sbws generate --torflow-round-digs`, you get:
`sbws generate: error: argument -r/--round-digs/--torflow-round-digs: expected one argument`

Reply:
Oh, sorry, you were right, got confused here.
I couldn't revert just those lines because would lose other changes.
I created a fixup that reverts those lines, and use torflow's rounding method in the test.
And added a new test for those options.